garrysmod/gamemodes/terrortown/gamemode/init.lua

```diff
@@ -207,7 +207,7 @@ function GM:Initialize()
    end
 
    local cstrike = false
-   for _, g in pairs(engine.GetGames()) do
+   for _, g in ipairs(engine.GetGames()) do

```

Comment:
This section can be changed to use https://wiki.garrysmod.com/page/Global/IsMounted